### PR TITLE
Update library packages for the extension

### PR DESCRIPTION
**Pull request by nishitasinha24**
Hope this looks okay. Thanks for reviewing.

### CODE REVIEW

**Comment by karthiknadig** [comment]
@nishitasinha24 Can you use python3.7 when creating your virtual environment?

File: dev-requirements.txt [hunk 1]
```diff
@@ -1,57 +1,47 @@
 #
-# This file is autogenerated by pip-compile with Python 3.7
+# This file is autogenerated by pip-compile with Python 3.11
```
Comment (karthiknadig) [inline dev-requirements.txt:2]: Looks like you have used python 3.11 to create the virtual environment where you ran `nox`. Can you do the same steps using python 3.7?

Comment (nishitasinha24) [reply dev-requirements.txt:2]: @karthiknadig I am getting an error while installing python 3.7.

Comment (nishitasinha24) [reply dev-requirements.txt:2]: @karthiknadig Hey I tried installing Python 3.7 again but it's still not working. 

Screenshot 2023-09-23 at 15 32 56

Comment (karthiknadig) [reply dev-requirements.txt:2]: you have an activated virtual environment. That environment might be 3.11. You will need to delete the `.venv` directory and then re-create the `.venv` using the `Python: Create Environment` command.

**Comment by karthiknadig** [comment]
Closing this for now. You can re-try a new PR, just use python 3.8 when creating the virtual environment.

**Pull request closed without merging**